docs: Add ES|QL drilldown support to dashboard and ES|QL visualizations pages (#5971)
elastic/docs-content: explore-analyze/dashboards/drilldowns.md

Reviewer: ```suggestion
For example, **Single click** has `{{event.value}}` and **Range selection** has `{{event.from}}` and `{{event.to}}`.

{applies_to}`stack: 9.4` {applies_to}`serverless:` {{esql}} visualization panels also support URL drilldowns.
```

![Drilldown on data table that navigates to another dashboard](/explore-analyze/images/kibana-dashboard_drilldownOnDataTable_8.3.gif "")

The following panel types support dashboard drilldowns:

* **Visualizations created with Lens**

> * {applies_to}`stack: 9.4` {applies_to}`serverless:` **{{esql}} visualizations**

Reviewer: ```suggestion
* {applies_to}`stack: ga 9.4` {applies_to}`serverless:` **{{esql}} visualizations**
```

* **Aggregation-based** area chart, data table, heat map, horitizontal bar chart, line chart, pie chart, tag cloud, and vertical bar chart

> * **Timelion**

Reviewer: ```suggestion
The following panel types support dashboard drilldowns:

* **Visualizations created with Lens**
* {applies_to}`stack: 9.4` {applies_to}`serverless:` **{{esql}} visualizations**
* **Maps**
* **TSVB**
* **Vega**
* **Aggregation-based** area chart, data table, heat map, horitizontal bar chart, line chart, pie chart, tag cloud, and vertical bar chart
* **Timelion**
```

:::{note}
:applies_to: { "stack":"ga 9.4", "serverless":"ga" }
**{{esql}} visualization** panels also support dashboard and URL drilldowns.
:::

> 

Reviewer: ```suggestion
```

* **Aggregation-based** area chart, data table, heat map, horitizontal bar chart, line chart, pie chart, tag cloud, and vertical bar chart
* **Timelion**

::::{note}
```{applies_to}
stack: ga 9.4
serverless: ga

> ```

Reviewer: :applies_to: { "stack":"ga 9.4", "serverless":"ga" }

> **Lens** panels that use ES|QL queries also support dashboard and URL drilldowns.

Reviewer: ```suggestion
**{{esql}} visualization** panels also support dashboard and URL drilldowns.
```

For example, **Single click** has `{{event.value}}` and **Range selection** has `{{event.from}}` and `{{event.to}}`.

> {applies_to}`stack: 9.4` {applies_to}`serverless:` {{esql}} visualization panels also support URL drilldowns.

Reviewer: ```suggestion
{applies_to}`stack: ga 9.4` {applies_to}`serverless:` {{esql}} visualization panels also support URL drilldowns.
```

![Drilldown on bar vertical stacked chart that navigates to Discover](/explore-analyze/images/kibana-dashboard_discoverDrilldown_8.3.gif "")

::::{note}

> Discover drilldowns are supported only by **Lens** panels that use data views. {{esql}}-based **Lens** panels do not support Discover drilldowns. To open all of the **Lens** dashboard panel data in **Discover**, check [Open panel data in Discover](../visualize/manage-panels.md#explore-the-underlying-documents).

Reviewer: ```suggestion
Discover drilldowns are supported only by **Lens** panels that use data views. {{esql}} visualization panels do not support Discover drilldowns. To open all of the **Lens** dashboard panel data in **Discover**, check [Open panel data in Discover](../visualize/manage-panels.md#explore-the-underlying-documents).
```

![Drilldown on bar vertical stacked chart that navigates to Discover](/explore-analyze/images/kibana-dashboard_discoverDrilldown_8.3.gif "")

::::{note}

> Discover drilldowns are supported only by **Lens** panels that use data views. ES|QL-based **Lens** panels do not support Discover drilldowns. To open all of the **Lens** dashboard panel data in **Discover**, check [Open panel data in Discover](../visualize/manage-panels.md#explore-the-underlying-documents).

Reviewer: ```suggestion
Discover drilldowns are supported only by **Lens** panels that use data views. {{esql}}-based **Lens** panels do not support Discover drilldowns. To open all of the **Lens** dashboard panel data in **Discover**, check [Open panel data in Discover](../visualize/manage-panels.md#explore-the-underlying-documents).
```
